### PR TITLE
Mainnet Configurability

### PR DESCRIPTION
**Pull request by BurntVal**

### CODE REVIEW

File: apps/abstraxion-dashboard/components/AbstraxionWallets/index.tsx [hunk 1]
```diff
@@ -100,7 +101,7 @@ export const AbstraxionWallets = () => {
       const signArbMessage = Buffer.from(encoder.encode(abstractAccount?.id));
       // @ts-ignore - function exists in keplr extension
       const signArbRes = await keplr.signArbitrary(
-        testnetChainInfo.chainId,
+        testnetChainInfo.chainId, // TODO: Make this responsive...or might be able to just ""
```
Comment (BurntVal) [inline apps/abstraxion-dashboard/components/AbstraxionWallets/index.tsx:104]: Here

File: apps/abstraxion-dashboard/components/AbstraxionSignin/index.tsx [hunk 2]
```diff
@@ -93,7 +93,7 @@ export const AbstraxionSignin = () => {
       return;
     }
     suggestAndConnect({
-      chainInfo: testnetChainInfo,
+      chainInfo: testnetChainInfo, // can't have chainInfo object as env var...
```
Comment (BurntVal) [inline apps/abstraxion-dashboard/components/AbstraxionSignin/index.tsx:96]: Here

File: apps/abstraxion-dashboard/components/AbstraxionWallets/index.tsx (hunk 1 above)
Comment (BurntVal) [reply apps/abstraxion-dashboard/components/AbstraxionWallets/index.tsx:104]: Empty string doesn't work even if signArb isn't using chainId

**Review by BurntVal — COMMENTED** [review]
Most of this is ready, just have a design question for Keplr/ChainInfo. See comments below:

File: apps/abstraxion-dashboard/components/AbstraxionSignin/index.tsx (hunk 2 above)
Comment (justinbarry) [reply apps/abstraxion-dashboard/components/AbstraxionSignin/index.tsx:96]: Could we use a chain name in the env var? You could also JSON.stringify the config into a env var.

Comment (BurntVal) [reply apps/abstraxion-dashboard/components/AbstraxionSignin/index.tsx:96]: Keplr requires the full chainInfo object when connecting. We can stringify the chainInfo, just wanted to see if there were other opinions with reasons I didn't see

**Review by BurntNerve — APPROVED** [review]
perfecto

**Pull request merged**